server/routes/createGoal/createGoalFormToCreateGoalDtoMapper.test.ts

```diff
@@ -0,0 +1,47 @@
+import type { AddStepDto, CreateGoalDto } from 'dto'
+import type { AddNoteForm, AddStepForm, CreateGoalForm } from 'forms'
+import moment from 'moment'
+import CreateGoalFormToCreateGoalDtoMapper from './createGoalFormToCreateGoalDtoMapper'
+
+describe('createGoalFormToCreateGoalDtoMapper', () => {
+  it.skip('should map to CreateGoalDto given valid form data', () => {
+    // Given
+    const createGoalDtoMapper = new CreateGoalFormToCreateGoalDtoMapper()
```

Comment:
I'll create a const for this before the test, like with `educationAndWorkPlanService` in educationAndWorkPlanService.test below.